developer_setup/classic_ui_split.md

```diff
@@ -14,7 +14,7 @@ But if you do..
 1. `git clone git@github.com:YOU/manageiq-ui-classic`
 1. `cd manageiq-ui-classic`
 1. `git remote add upstream git@github.com:ManageIQ/manageiq-ui-classic.git`
-1. `ln -s ../../manageiq spec/`
+1. `ln -s ../manageiq spec/`
```

Comment:
Man page on MacOS is different. Therefore you don't see the excerpt from `man ln` that I mentioned previously.